src/components/ScreenTemplates.vue

```diff
@@ -1,12 +1,32 @@
 <template>
-  <div>
+  <div data-cy="screen-templates-section">
     <div class="d-flex justify-content-between">
       <h6 class="pt-2">Select a Template</h6>
```

Comment:
Can we wrap these hardcoded texts in the translations method `{{ $t("Select a Template") }}`

Reply:
Will be resolved in https://github.com/ProcessMaker/processmaker/pull/7369